stories/core/DropdownMenu.stories.tsx

```diff
@@ -314,3 +318,52 @@ export const WithSeparator: Story<DropdownMenuProps> = (args) => {
 WithSeparator.decorators = [
   (Story) => <div style={{ minHeight: 200 }}>{Story()}</div>,
 ];
+
+export const WithContent: Story<DropdownMenuProps> = (args) => {
+  const { menuItems, ...rest } = args;
+
+  const onClick = (item: string, close: () => void) => () => {
+    action(`'${item}' clicked!`)();
+    close();
+  };
+
+  const [userType, setUserType] = useState('User');
+
+  const dropdownMenuItems = (close: () => void) => [
+    <MenuContent key={0}>
+      <>
+        <Text variant='leading'>Terry Rivers</Text>
+        <Text isMuted style={{ marginBottom: 8 }}>
+          terry.rivers@email.com
+        </Text>
+        <Select
```

Comment:
This Select gets focused using arrow keys. Seems wrong to me as it's not a list item.

Reply:
I think when menu opened, first focusable element should be focused (even if its in non interactive menu item)..no??

Reply:
Created an issue: https://github.com/iTwin/iTwinUI-react/issues/379